test/strategy_test.rb

```diff
@@ -75,7 +75,7 @@ class AuthorizeParamsTest < StrategyTestCase
 
 class TokeParamsTest < StrategyTestCase
   test 'has correct parse strategy' do
-    assert_equal :query, strategy.token_params[:parse]
+    assert_equal nil, strategy.token_params[:parse]
   end
 end
```

Comment:
@printercu I think we could remove this entire `TokeParamsTest` now (lol that typo btw. :))